core/Updates/2.14.1-b2.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Updates;
+
+use Piwik\Common;
+use Piwik\Updater;
+use Piwik\Updates;
+
+/**
+ * Update for version 2.14.0.
+ */
+class Updates_2_14_1_b2 extends Updates
+{
+    /**
+     * Here you can define one or multiple SQL statements that should be executed during the update.
+     * @return array
+     */
+    static function getSql()
+    {
+        $updateSql = array(
+            'ALTER TABLE `' . Common::prefixTable('site') . '` ADD COLUMN `exclude_unknown_urls` TINYINT(1) DEFAULT 0 AFTER `currency`' => array(1060)
```

Comment:
getSql() was deprecated in 2.12.0 - use getMigrationQueries() instead

See example of new use case in https://github.com/piwik/piwik/blob/master/core/Updates/2.14.0-b2.php#L25-24

Ideally we would update all old core/Updates/* to have consistent use of this new function (feel free to do it in new PR if you want)